app/assets/javascripts/admin/addon/components/admin-permalink-form.gjs

```diff
@@ -45,7 +45,7 @@ export default class AdminFlagsForm extends Component {
       } else if (!isEmpty(this.args.permalink.category_id)) {
         permalinkType = "category";
         permalinkValue = this.args.permalink.category_id;
-      } else if (!isEmpty(this.args.permalink.tag_name)) {
+      } else if (!isEmpty(this.args.permalink.tag_id)) {
```

Comment:
Permalinks store `tag_id`, not `tag_name`. If `tag_id` is set, but the tag can't be found, this check would render the permalink as "Topic permalink" instead of "Tag permalink". Which might be unexpected for users.